includes/discovery/mempools/bdcom.inc.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/*
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+
+ * @package    LibreNMS
+ * @subpackage discovery
+ * @link       http://librenms.org
+ * @copyright  2017 - Carlos A. Pedreros Lizama <carlos.pedreros@gmail.com>
+ * @author     Carlos A. Pedreros Lizama <carlos.pedreros@gmail.com>
+
+ .1.3.6.1.4.1.3320.9.48.1 nmsMemoryPoolUtilization.0 = [%][This is the memory pool utilization"]
```

Comment:
Please remove all the comments here, this is all in the mib `mibs/pbn/NMS-MEMORY-POOL-MIB.MIB`